C/c4PredictiveQuery.cc

```diff
@@ -59,15 +60,23 @@ class C4PredictiveModelInternal : public PredictiveModel {
     C4PredictiveModel _c4Model;
 };
 
+#endif // COUCHBASE_ENTERPRISE
+
 
 void c4pred_registerModel(const char *name, C4PredictiveModel model) C4API {
+#ifdef COUCHBASE_ENTERPRISE
     auto context = retained(new C4PredictiveModelInternal(model));
     context->registerAs(name);
+#else
+    abort();
```

Comment:
when fails, should we need to include an error message for easy debugging?

Reply:
ok.